gatsby-node.ts

```diff
@@ -10,3 +11,110 @@ export const onCreateWebpackConfig: GatsbyNode['onCreateWebpackConfig'] = ({
     },
   });
 };
+
+function checkRelativePaths(obj: any): boolean {
+  const files = [`AgencyInfo.csv`, `DeliverInvest.csv`];
+  for (const edge of obj.data.allFile.edges) {
+    if (!files.includes(edge.node.relativePath)) {
+      return false;
+    }
+  }
+  return true;
+}
+
+exports.createPages = async ({ graphql, actions }) => {
+  const { createPage } = actions;
+
+  const relativePaths = await graphql(
```

Comment:
The application expects certain files to exist. This first query check to see if those files exist. If it doesn't throw and error that super obvious. If it does, continue normal operation.

This might have to fail a little more gracefully .... in production?